### PR TITLE
RIFEX-154 Refresh network data in all cases

### PR DESCRIPTION
**Pull request by debrayperalta**
This PR includes #76 , please merge it first

This was an issue with the tokenAddress and checksumming when searching the network data, so I lowercased all the addresses so the search will come as successful.

This solves RIFEX-154

### CODE REVIEW

File: ui/app/rif/actions/index.js [hunk 1]
```diff
@@ -892,7 +892,7 @@ function getLuminoNetworkData (tokenAddress) {
         userChannels: 0,
       }
       dispatch(this.getTokens()).then(tokens => {
-        const data = tokens.find(n => n.address === tokenAddress);
+        const data = tokens.find(n => n.address.toLowerCase() === tokenAddress.toLowerCase());
```
Comment (JONAF2103) [inline ui/app/rif/actions/index.js:894]: I don't know if you need to do this, seems to me the addresses should be all checksumed, anyway it doesn't make any harm to check on lowecase i guess.

File: ui/app/rif/components/luminoNetworkChannels.js [hunk 2]
```diff
@@ -97,21 +97,22 @@ class LuminoNetworkChannels extends Component {
...
           </div>
-      }
-      {
-        data.length === 0 &&
-        <div>
-          <span className={classes.title}>Lumino Channels</span>
-          <span className={classes.notFound}>No channels found</span>
-        </div>
-      }
+        }
```
Comment (JONAF2103) [inline ui/app/rif/components/luminoNetworkChannels.js]: Is this 2 or 4 identation spaces? be careful with that, specially if you use visua studio as an IDE.

**Review by JONAF2103 — APPROVED** [review]
LGTM i just put some comments there.

File: ui/app/rif/components/luminoNetworkChannels.js (hunk 2 above)
Comment (debrayperalta) [reply ui/app/rif/components/luminoNetworkChannels.js]: I am using Webstorm, so it is 2 identation spaces. (I believe)

File: ui/app/rif/actions/index.js (hunk 1 above)
Comment (debrayperalta) [reply ui/app/rif/actions/index.js:894]: I dunno what happened on this case, but on one of the flows it worked and in the other didn't.

So for being sure I lowercased it,

**Review by marcosmartinez7 — APPROVED** [review]

**Pull request merged**